theseus/core/cost_weight.py

```diff
@@ -13,6 +13,9 @@
 from .variable import Variable, as_variable
 
 
+_ZERO_EPS = 1.0e-15
```

Comment:
Could move to constants.py if this is outside of the class?

Reply:
This is not used anymore.